sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
-
+## 1.0.9 (2020-07-15)
```

Comment:
You'll need to change the CHANGELOG date to today otherwise the release pipeline will fail. I'd merge this PR in first and push the update directly to the branch.

Reply:
@alzimmermsft, I'm not sure that's the case. @chidozieononiwu updated the changelog verification to only check that there is a date.

Reply:
Yes @JimSuplizio is right. We loosened up the check so as long as it is a valid date it will work.

Reply:
Would it fail if set to a future date? We were planning to ship on Friday but not sure if that will fail the pipeline too.